packages/libs/wrapped-keys/src/lib/lit-actions/lit-actions-global.d.ts

```diff
@@ -0,0 +1,365 @@
+import { ethers as Ethers } from 'ethers';
+
+declare global {
```

Comment:
See other comment on replacing this, which mutates `global` for our entire ts project (all packages, all scopes), with explicit declarations like we did in the starter-kit repo.

This can be a follow-up ticket, as I am sure we've got this global d.ts file in multiple places in the repo by now...